### DISCUSSION TITLE
Weird output message when using `brew search`

### DISCUSSION THREAD

**Discussion by RoamanEmpire** [Homebrew/discussions#1304] [Everyday usage]
Whenever I use the `brew search` command lately I always get this message printed before the command output:
`Going to report it - RyotaK (https://hackerone.com/ryotak)`

Here's an example screenshot:
![homebrew](https://user-images.githubusercontent.com/69714537/115142234-f2652c80-a040-11eb-8868-0f933f5ba0c4.png)

This is the version of Homebrew I'm using.
```
~ brew --version
Homebrew 3.1.2
Homebrew/homebrew-core (git revision 0218118076; last commit 2021-04-18)
Homebrew/homebrew-cask (git revision 97ad5dda47; last commit 2021-04-18)
```

Am I the only one? Is this normal?

**Comment by stonerl** [comment]
I have this message as well when using `brew upgrade` or `brew list`

```
➜  ~ brew upgrade
Going to report it - RyotaK (https://hackerone.com/ryotak)
```

**Comment by stonerl** [comment]
I assume this guy found a vulnerability in Homebrew and is using it to tell people about? Just a guess though.

**Comment by stonerl** [comment]
Have you installed iTerm2?

https://github.com/Homebrew/homebrew-cask/commit/97ad5dda47f9a15aec5c397ef3b4192422bed1b3#comments

This commit seems to be the culprit.

**Reply by RoamanEmpire** [reply]
Looks like that is the source of the message, however I didn't install iTerm2 which makes this even weirder.
Perhaps there's another cask out there with the same embedded message.

**Comment by stonerl** [comment]
Uninstall the iTerm2 Cask and the message is gone.

**Comment by al-bonin** [comment]
I am done with HomeBrew!!! He could literally install/exchange iTerm2 with any malware and I would put it myself in my $PATH thanks to HomeBrew. I am just done with Brew. Just give me a reason not to.

**Comment by RoamanEmpire** [comment]
Looks like this was approved by the Homebrew staff: https://github.com/Homebrew/homebrew-cask/pull/104191
So perhaps there is nothing to worry about

**Comment by stonerl** [comment]
If they gonna do such tests maybe a warning to the users would be helpful.
